orchestration/dagster_orchestration/hca_utils/main.py

```diff
@@ -7,7 +7,7 @@
 
 from hca_utils import __version__ as hca_utils_version
 from .utils import HcaUtils
-from hca_orchestration.base import default_google_access_token
+from hca_orchestration.sensors import default_google_access_token
```

Comment:
:-1: this is an import from a file that imports it, let's import directly from the file where this method's defined instead:
```suggestion
from hca_orchestration.resources.base import default_google_access_token
```